ecommerce-ui/src/App.css

```diff
@@ -0,0 +1,48 @@
+.flex-container {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: -webkit-box;
```

Comment:
For future reference, `create-react-app` adds these prefixes automatically 😄. No need to add them, which is pretty cool. https://github.com/facebook/create-react-app#whats-included